### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- Add a new comment 
- Delete comment 
- minor changes

### CODE REVIEW

**Review by parulgupta2001 — APPROVED** [review]

File: src/features/posts/components/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        <Button
+          component={Link}
+          to="/profile"
+          sx={{
+            borderRadius: "999px",
+          }}
+        >
+          <Avatar
+            sx={{
+              bgcolor: "#1565C0",
```
Comment (parulgupta2001) [inline src/features/posts/components/Comment.jsx:66]: use var for color

**Review by parulgupta2001 — COMMENTED** [review]
small suggestion otherwise everything looks good.

**Review by sharathnair9999 — APPROVED** [review]
Liked how minimal u r keeping the PR. Good work

File: src/features/posts/components/Comment.jsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/features/posts/components/Comment.jsx:66]: I am using materila ui

**Pull request merged**